src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.targets

```diff
@@ -379,15 +379,10 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!-- Determine what compiler and versions of the language to target. For 2.x targeting projects, these are carried by packages referenced by the project. Use this -->
 
-  <!-- Use the 2.x compiler if we're building an application that references Microsoft.AspNetCore.Razor.Design. -->
-  <Import Project="$(RazorCodeGenerationTargetsPath)"
```

Comment:
Not using the code generation targets. Fortunately enough of the details about what RazorConfiguration to use is defined in the $Package.props so that the SDK's copy of CodeGeneration.targets just works.